assembl/static2/css/views/home.scss

```diff
@@ -377,6 +384,24 @@
 }
 
 @media screen and (max-width: $screen-sm-max) {
+  .home {
+    .objectives-section {
+      .objectives-img {
+        &.margin-right {
```

Comment:
Picky, but possible rename? Also, the objectives section has 6 different variations, based on mockups. You should perhaps encompass this in multiple css classes, one for each layout configuration. Unless this style is global across all configurations of objective layouts

Reply:
See 8de691d39fa7ee78ce240272757354ce7a745174 branch for sample layout

Reply:
@ayyazdaniaryan I don't understand what you meant in here. Could we spend sometime tomorrow after the stand up to talk about it ? Thx.

Reply:
Yeah sure, we can discuss it. But in general, what I mean is, you have `objectives-section` class as the encompassing class, then suddenly into `objectives-img` class. When there are multiple configurations, this css class will break. So, create a class now, `objectives-config-1` that encompasses the first of the 6 configs, and then update the css, such as margins etc. This way, the css changes on [8de691d](https://github.com/assembl/assembl/commit/8de691d39fa7ee78ce240272757354ce7a745174) won't be drastic. They'll just use `objectives-config-${configurationNumber}` in the css class of the view.

Reply:
just added an other layer of abstraction: from our conversation - this is a temp solution to meet different needed configuration since objectives component should be rebuild from scratch